packages/backend/src/controllers/authentication/strategies/azureStrategy.ts

```diff
@@ -51,13 +51,12 @@ const azureAdPrivateKeyJksStrategy = async (): Promise<
      * certificate SHA-1 thumbprint, base64-encoded, for purposes of key
      * matching.
      *
-     * Confusingly, it may also complain about the x5t and x5c claims, but we
-     * can ignore it here and simply override whatever value we have from the
-     * underlying private key, and node-openid-client will know to include it
-     * as part of the jwt header.
+     * The Azure documentation is a bit inconsistent in this regard, but if
+     * we override whatever the `kid` claim is at this point, openid-client
```

Comment:
This comment is important but belongs in `buildJwtKeySet` as the comment below suggests!